dali/base/dafdesc.cpp

```diff
@@ -3701,9 +3701,14 @@ class CStoragePlaneInfo : public CInterfaceOf<IStoragePlane>
     {
         return xml->queryProp("storageapi/@account");
     }
-    virtual const char * queryStorageContainer() const override
+    virtual StringBuffer & getStorageContainer(StringBuffer & container, unsigned stripeNumber) const override
     {
-        return xml->queryProp("storageapi/@container");
+        StringBuffer path("storageapi/");
+        if (stripeNumber==0)
+            path.append("@container");
+        else
+            path.appendf("containers/@stripe%d", stripeNumber);
+        return container.append(xml->queryProp(path.str()));
```

Comment:
Should this throw an error if the container is not defined in the correct way.  Can the helm chart check for some consistency - e.g., number of containers matches the number of devices?

Reply:
I'll get helm chart to check the configuration and add a check here too.